src/main.js

```diff
@@ -22,54 +22,49 @@ function buildHeader(apiKey) {
     'Content-Type': 'application/json',
     'x-api-key': apiKey,
     'anthropic-version': '2023-06-01',
-    'anthropic-beta': 'messages-2023-12-15',
```

Comment:
Sorry for reviewing this PR only now. After checking the latest Anthropic API documentation, I think maybe the value of "anthropic-beta" should be updated to "max-tokens-3-5-sonnet-2024-07-15" to align with the most recent changes?

Reply:
Anthropic mentions in https://docs.anthropic.com/en/api/versioning that 'anthropic-version' is a required parameter to be passed, while 'anthropic-beta' seems to be an old value that has been deprecated